Domain/repositoriesInterface.go

```diff
@@ -1,4 +1,7 @@
 package Domain
 
 type RepositoryInterface interface {
```

Comment:
ClientRepositoryInterface ? It's just an idea, we could refactor when there is need to add one more repo into project